src/applications/discharge-wizard/utilities/display-logic-questions.js

```diff
@@ -32,10 +32,12 @@ export const displayConditionsMet = (SHORT_NAME, formResponses) => {
 
   if (questionRequirements.includes('FORK')) {
     const forkedReqs = displayConditionsForShortName?.FORK;
+    let anyForkConditionsMet = false;
```

Comment:
I realized we do need these vars for the skip ahead questions and checking for a valid DC